src/models/eventModel.tsx

```diff
@@ -19,6 +19,10 @@ const ctx = {
   hasFailureRate: VocabularyUtils.PREFIX + "has-failure-rate",
   requirement: VocabularyUtils.PREFIX + "has-requirement",
   upperBound: VocabularyUtils.PREFIX + "to",
+  criticality: VocabularyUtils.PREFIX + "criticality",
+  hasPrediction: VocabularyUtils.PREFIX + "has-prediction",
+  value: VocabularyUtils.PREFIX + "value",
+  behavior: VocabularyUtils.PREFIX + "behavior",
```

Comment:
This should be:
 behavior: VocabularyUtils.PREFIX + "**is-manifestation-of**"

Reply:
@kostobog but i guess it makes sens to have something like:
isManifestationOf: VocabularyUtils.PREFIX + "is-manifestation-of",


I mean, I do not see reason to have different json name and RDF property.

Reply:
@blcham I think in json or jopa object `behavior` is more readable than `isManifestationOf`. May be we can change it in the future.

Reply:
so, i'll keep "behavior", right ?